source/localizable/cps_orders_api/index.html.md.erb

```diff
@@ -0,0 +1,128 @@
+---
+title: cps_orders_api
+---
+
+<%= partial 'partials/page_locales' %>
+
+# Orders API <%= edit_link %>
+
+Orders API allows Skroutz Merchants to perform actions on their CPS orders,
+such as accepting or rejecting an order.
+
+<%= partial 'partials/toc' %>
+
+## Setup
+
+### Obtaining an API token
+
+In order to be able to use the API, you need to generate an **API token** from within CPS orders settings
+page in merchant's panel (Merchants > Παραγγελίες καταστήματος > CPS orders API).
+
+> <i class="fa fa-exclamation-triangle"></i>
+> There can only be one active **API key** per shop at a certain time.
+> Creating a new API key will expire any previous one.
+
+### Authorization
+
+Pass in a valid **API token** in the `Authorization` header, prefixed with `Bearer `
+as in the following curl example.
+
+A specific `Accept` header is also required. For more information on API requests check the [API overview](/api/v3/).
+
+<pre class="terminal">
+curl -X GET https://<%= settings.api_domain %>/merchants/cps/orders/1234567 \
+  -H 'Accept: application/vnd.<%= settings.site_name %>+json; version=3.0' \
+  -H 'Authorization: Bearer your_access_token_here'
+</pre>
+
+## Endpoints
+
+
+### Fetch a single order
+
+To fetch a specific order, you have to provide the order `code` with which you have originally identified 
+your order in your shop, as a URL parameter.
+
+When an order is in a state that can be rejected, `reject_options` attribute is included in the
+response, providing all the available options for issuing "reject" requests for the order.
+
+<pre class="terminal">
+  GET /merchants/cps/orders/:code
+</pre>
+
+<%= render_recording :merchants_cps_orders_show %>
+
+
+### Reject a single order
+
+#### Reject order with a predefined reason
+
+In order to reject an order, you **must** include a reject option.
+
+The [available](/cps_orders_api/#reject-options) rejection reason names are provided in `reject_options` when retrieving a single order.
+
+<pre class="terminal">
+  POST /merchants/cps/orders/:code/reject
+</pre>
+
+<%= render_recording :merchants_cps_orders_reject %>
+
+##### Example
+<pre class="terminal">
+curl -X POST https://<%= settings.api_domain %>/merchants/cps/orders/1234567/reject \
+  -H 'Accept: application/vnd.<%= settings.site_name %>+json; version=3.0' \
+  -H 'Content-Type: application/json; charset=utf-8' \
+  -H 'Authorization: Bearer your_access_token_here' \
+  -d '{ "reason": { "name": "user_cancel" } }'
+</pre>
+
+
+#### Reject order with `other` reason
```

Comment:
I wouldn't mention `other` here. Instead I'd replace it with `custom`:

> Reject order with custom reason

In the next paragraph, you explain very well what is needed and what `other` is.